docs/guide/authentication.md

```diff
@@ -185,7 +185,7 @@ Dozzle supports multi-user authentication by setting `--auth-provider` to `simpl
 ```yml
 users:
   admin:
```

Comment:
@sureshram this line is actually the username.

Line 188 is the display name. Like "John Smith"

Reply:
I see, The confusion is that, it's not clear which one is the username field. "admin" sounds like a role. Added commit make it more clear.